test/functional/p2p_segwit.py

```diff
@@ -1297,8 +1321,6 @@ def test_tx_relay_after_segwit_activation(self):
         # Node will not be blinded to the transaction
         self.std_node.announce_tx_and_wait_for_getdata(tx3)
         test_transaction_acceptance(self.nodes[1], self.std_node, tx3, True, False, 'tx-size')
-        self.std_node.announce_tx_and_wait_for_getdata(tx3)
```

Comment:
I got really confused reviewing master's test. The surrounding test is dead, with incorrect comments asserting that we are checking for blinding. Seems this was deleted just to make the test pass :grimacing: